Šumava map changes

Syrrealll opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [X] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:
Šumava is not a small map, being 151km2 in size, coming close to maps like Chernarus or Livonia. Yet compared to these maps, which have 44 and 51 POIs respectively, šumava has a puny 30, less than even Malden (32) or Takistan (34).

So I went and added the following to the emptier parts of the map:

- 2 mil bases
- 8 resources
- 8 outposts (1 split from airport_2)

Bringing the total up to 48, on par with other maps this size.

The only change in existing map markers was shortening the egregious 2 kilometers long airport_2 marker to somewhat normal size, and making the other end into outpost_20.
No markers were removed, no markers were changed besides the one stated above.


### Please specify which Issue this PR Resolves (If Applicable).

### Please verify the following.

1. [X] Have you loaded the mission in LAN host?
2. [X] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:
I've checked that the map loads and works, POIs spawn in correctly, with flags, vehicles, supply boxes. However, I have not played a full campaign with this, so I don't know how it would feel when played.

Syrrealll committed:
Add files via upload

stutpip123 commented:
how many resources does the map now have in total?

Syrrealll commented:
It had 3 before, so the total is 11 resources to 7 factories now. On par with Livonia (12 resources to 9 factories) or Cherno (9 to 5).
If I could I would remove a factory or two I would but alas.

stutpip123 committed:
Merge branch 'unstable' into unstable

stutpip123 reviewed (commented):
Took it into editor, looks mostly fine. Outpost 9 at the bottom left might need resizing due to the large cargo tower being slightly outside it. Probably ready for playtesting

stutpip123 committed:
Fix Zone Sizes

sorted out sizes of many zones to ensure static weapon and garrison-able buildings are within them
